### ISSUE TITLE
add aria-label on social links 

### ISSUE THREAD

**Issue by beatricelucaci** [WPChill/modula-lite#919]
**Is your feature request related to a problem? Please describe.**
I work very hard on my sites to try and make them technically correct. I use Google Lighthouse through the inspector to run checks and fix issues. When I run this check Modula gallery fails for the links it creates and also for accessibiilty so the score is reduced.

I have to say I am a bit disappointed as I would not expect to pay for a premium plugin and find these errors. You can even check them on your website as they also appear there. Now the link warning seems to potentially be a false positive but the accessibility issue is not as the aria attributes are missing.

https://support.wpchill.com/conversation/11941?folder_id=81
